ttnn/ttnn/__init__.py

```diff
@@ -465,6 +465,7 @@ def manage_config(name, value):
     logaddexp_bw,
     logaddexp2_bw,
     squared_difference_bw,
+    remainder_bw,
```

Comment:
This file should be removed while rebasing, Please have a check on this once you rebase with main